tests/test_fx/test_codegen/test_activation_checkpoint_codegen.py

```diff
@@ -49,16 +49,20 @@ def __init__(self):
         self.relu = relu()
         self.linear2 = torch.nn.Linear(4, 4)
 
+    def ckpt2(self, x):
+        return F.relu(x, inplace=True)
+
+    def ckpt3(self, x):
+        return self.linear2(x)
+
     def forward(self, x):
```

Comment:
Shall we try the multiple inputs test here? For example,
```python
def forward(self, x1, x2):
      y1 = checkpoint(self.layer1, x1)
      y2 = checkpoint(self.layer2, x2, y1)
      return y1 + y2
```

Reply:
Yeah, I think it's better, I will add it later

Reply:
I've added tests as you suggested